server/cmd/mmctl/commands/user.go

```diff
@@ -736,26 +736,31 @@ func deleteUsersCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	var errs *multierror.Error
 	users, err := getUsersFromArgs(c, args)
 	if err != nil {
-		printer.PrintError(err.Error())
+		errs = multierror.Append(errs, err)
 	}
```

Comment:
Here we can just return directly if there is an error. We can't recover from that.